public/locales/es/common.json

```
@@ -9,5 +9,6 @@
     "moreInformation": "Más información",
     "jumpTo": "Salta a",
     "aqiHelpHeading": "¿Qué es AQI?",
-    "aqiHelpMessage": "AQI es el índice de la calidad del aire. Es el estándar nacional para indicar si el aire está bien o mal. Los números más bajos indican aire más saludable, mientras que los números más altos indican aire más peligroso."
+    "aqiHelpMessage": "AQI es el índice de la calidad del aire. Es el estándar nacional para indicar si el aire está bien o mal. Los números más bajos indican aire más saludable, mientras que los números más altos indican aire más peligroso.",
+    "noSensorGauge": "¡Haz clic en un sensor en el mapa para ver el Índice de la calidad de aire (Air Quality Index, AQI) aquí!"
```

In Spanish, the message is pretty long, so I think we can say this since the question mark is right next to it and gives AQI a full definition there
```suggestion
    "noSensorGauge": "¡Haz clic en un sensor en el mapa para ver el AQI aquí!"
```